app/controllers/api/v1/locations_controller.rb

```diff
@@ -3,7 +3,7 @@ class API::V1::LocationsController < ApplicationController
 
   def index
     @locations = Location.includes(:experiences)
-      .where('experiences_count > ?', 0).all
+                 .where('experiences_count > ?', 0).all
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.